src/graph/executor/logic/LoopExecutor.cpp

```diff
@@ -20,6 +20,9 @@ folly::Future<Status> LoopExecutor::execute() {
   QueryExpressionContext ctx(ectx_);
 
   auto value = expr->eval(ctx);
+  if (value.isNull()) {
+    value = Value(true);
```

Comment:
it seems that u should assign false to the value. maybe you need not to check the result of value, just delete the following DCHECK.

Reply:
Removed.

Reply:
It won't pass the tck.